### PR TITLE
PACT-735 Added response header to prevent browser caching of pages

### PR DESCRIPTION
**Pull request by userjd**

### CODE REVIEW

File: app/uk/gov/nationalarchives/omega/editorial/support/filter/ResponseHeaderFilter.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+package uk.gov.nationalarchives.omega.editorial.support.filter
+
+import akka.stream.Materializer
+import play.api.Logging
+import play.api.mvc._
+
+import javax.inject.Inject
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ResponseHeaderFilter @Inject() (implicit val mat: Materializer, ec: ExecutionContext)
```
Comment (rwalpole) [inline app/uk/gov/nationalarchives/omega/editorial/support/filter/ResponseHeaderFilter.scala:31]: I think we should just call this `CacheControlFilter` as that is all it does currently. The actually filtering is done on the request header and the result is the response header. We are not filtering the response header as such.

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: app/uk/gov/nationalarchives/omega/editorial/support/filter/ResponseHeaderFilter.scala (hunk 1 above)
Comment (userjd) [reply app/uk/gov/nationalarchives/omega/editorial/support/filter/ResponseHeaderFilter.scala]: Done

File: app/uk/gov/nationalarchives/omega/editorial/support/filter/CustomFilter.scala [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+package uk.gov.nationalarchives.omega.editorial.support.filter
+
+import play.api.http.DefaultHttpFilters
+import play.filters.cors.CORSFilter
+import play.filters.csrf.CSRFFilter
+
+import javax.inject.Inject
+
+class CustomFilter @Inject() (
```
Comment (rwalpole) [inline app/uk/gov/nationalarchives/omega/editorial/support/filter/CustomFilter.scala:30]: Can we call this `CustomFilters` with an `s` as it is really a collection of filters.

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: app/uk/gov/nationalarchives/omega/editorial/support/filter/CustomFilter.scala (hunk 2 above)
Comment (userjd) [reply app/uk/gov/nationalarchives/omega/editorial/support/filter/CustomFilter.scala]: Done

**Review by rwalpole — APPROVED** [review]

**Pull request merged**